Src/BootCamp.Chapter/ILog.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    interface ILog
+    {
+        public void WriteMessageToFile(string messge);
+        public void WriteMessageToConsole(string messge);
```

Comment:
Purpose of an interface should provide a contract of what can be done.
You can log- that's what yuo can do.
ToFile or ToConsole- is a detail- not something that you should concern an interface with.

Reply:
Thanks for sharing! I have more sense of interface^_^